storage/innobase/include/ut0mutex.ic

```diff
@@ -54,13 +54,19 @@ TTASEventMutex<Policy>::wait(
 {
 	sync_cell_t*	cell;
 	sync_array_t*	sync_arr;
-
+	latch_id_t		latch_id = m_policy.get_id();
+	ulint			type = SYNC_MUTEX;
+	
```

Comment:
can you align latch_id and type at same level as "cell and sync_arr variables"

Reply:
done. This file uses tabs instead of spaces and that causes that formatting depends on tab width. I.e. in my editor where tab is set to 2 spaces now it looks like this:
![image](https://user-images.githubusercontent.com/50845928/72093127-1a30f500-3314-11ea-9a84-840b5fc09eb3.png)

Reply:
I think you should set it to 4. Check github page. It looks correct to me now